### DISCUSSION TITLE
Problem loading fonts in d3d

### DISCUSSION THREAD

**Discussion by etduxing** [community/community#65098] [Programming Help]
### Body

How can I get him to load the fonts in my system
![image](https://github.com/community/community/assets/95662186/bf921cfa-f797-4b75-b92e-3d2519cce8ff)

### Guidelines

- [X] I have read and understood this category's [guidelines](https://github.com/orgs/community/discussions/51384) before making this post.

**Comment by arafajs** [comment]
Make sure that the font files are in the correct location.
